Makefile

```diff
@@ -7,7 +7,13 @@ TAG = latest
 BUILD_OPTS = 
 
 build:
-	docker build $(BUILD_OPTS) -t $(IMAGE_NAME):$(TAG) .
+	@if [ -z "$$GH_TOKEN" ]; then \
+		echo "ERROR: GH_TOKEN environment variable is not set"; \
```

Comment:
what's the rationale for this part?

Reply:
My thinking was that the cborg-code repo is private, so you need a token to clone it [here](https://github.com/justaddcoffee/odk-ai/blob/69750a4578fb676451d1ea927575bc9ff3f675f3/Dockerfile#L64)

Reply:
ah, that makes sense